docs/learn/digital-assets/read-asset-data.md

```diff
@@ -267,744 +302,79 @@ console.log(assetMetadata);
 
-Some of the 🗂️ [ERC725Y data keys](../../standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store) feature dynamic inputs. Make sure to exchange `<myCreatorAddress>` with the actual address (without `0x`) of one of the asset's creators you want to retrieve. In case the asset does not feature any additional creator information, the returned value of `LSP4CreatorsMap:<myCreatorAddress>` will be `0`.
+Follow the guide [**Read Profile Data**](../universal-profile/metadata/read-profile-data.md) to retrieve the profile infos!
 
-Optionally, you can [retrieve the profile data](../universal-profile/metadata/read-profile-data.md) of a creator in a separate step.
 :::
 
-<!-- prettier-ignore-start -->
+The `LSP4CreatorsMap:<address>` is a Mapping data key that requires a dynamic input. To fetch its info, use the function below, providing the creator address as input to the `dynamicKeyParts` property of the object parameter.
 
 ```js
-// ...
-
-// Fetch the asset's token symbol
-const tokenSymbol = await myAsset.fetchData('LSP4TokenSymbol');
-console.log(tokenSymbol);
-
-// Fetch the asset's token name
-const tokenName = await myAsset.fetchData('LSP4TokenName');
-console.log(tokenName);
-
-// Fetch all creators of the asset
-const assetCreatorsList = await myAsset.fetchData('LSP4Creators[]');
-console.log(assetCreatorsList);
-
 // Fetch creator-specific information
-const creatorInformation = await myAsset.fetchData([
-  /* 
-   * for dynamic keys, it is necessary 
-   * to provide any second data key
-  */ 
-  'LSP4TokenName',
+const creatorInformation = await erc725js.fetchData([
   {
     keyName: 'LSP4CreatorsMap:<address>',
     // Sample creator address
     dynamicKeyParts: '0x9139def55c73c12bcda9c44f12326686e3948634',
   },
 ]);
 console.log(creatorInformation);
+// Dynamic Creator Map Retrieval
+/*
+ * Creator Map for 0x9..8634 (empty, no data set)
+ * Key Format: '0x6de85eaf5d982b4e5da00000<myCreatorAddress>'
+ * Name Format: 'LSP4CreatorsMap:<myCreatorAddress>'
+
+[
+  
+  {
+    key: '0x6de85eaf5d982b4e5da000009139def55c73c12bcda9c44f12326686e3948634',
+    name: 'LSP4CreatorsMap:9139def55c73c12bcda9c44f12326686e3948634',
+    value: [],
+  },
+];
+*/
 ```
 
-<!-- prettier-ignore-end -->
+## Interpret Data based on Token Type
 
-<details>
+If both, contract and metadata standard are aligned, Finally we need to determine the [LSP4 Token Type](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#types-of-digital-assets) in order to interpret the metadata correctly.
 
-```js
-// Creator List (empty, no creators set)
-{
-  key: '0x114bd03b3a46d48759680d81ebb2b414fda7d030a7105a851867accf1c2352e7',
-  name: 'LSP4Creators[]',
-  value: []
-}
+At this point, you should be able to identify if the digital asset is a:
 
-// Token Symbol (UPT)
-{
-  key: '0x2f0a68ab07768e01943a599e73362a0e17a63a72e94dd2e384d2c1d4db932756',
-  name: 'LSP4TokenSymbol',
-  value: 'UPT'
-}
+- LSP7 - Token
```

Comment:
Should we link these?

Reply:
We don't have the specific content yet and need to create this page **Choose LSP7 vs LSP8**. I would leave it like this for now. But good call!